docs/api/app.md

```diff
@@ -1426,19 +1426,6 @@ This is the user agent that will be used when no user agent is set at the
 app has the same user agent.  Set to a custom value as early as possible
 in your app's initialization to ensure that your overridden value is used.
 
-### `app.allowRendererProcessReuse`
```

Comment:
we should mark this as deprecated in 13.x. did it already make it into docs/breaking-changes as a deprecation?

Reply:
It's definitely been deprecated since like, Electron 11.  There's been a warning.  I'm not sure if we put deprecations in breaking-changes.  The default flip back in Electron 9 is in the breaking-changes list though.

Reply:
![image](https://user-images.githubusercontent.com/172800/115476303-7a406780-a1f6-11eb-8997-bfba76a1b184.png)

We generally mark deprecations in the docs for a feature when we deprecate it, and for major things we put them in breaking changes too.

Reply:
Hm, I'll put a PR back into 13-x-y to add the @depreacted tag